db/seeds.rb

```diff
@@ -147,92 +56,28 @@
-12674
-79394
-19860
-64350
-39193
-43777
-52376
-68211
-44764
-64304
-36353
-41204
-31701
-26684
-44040
-01744
-69863
-82543]
+user = User.new(name: "admin", email: "sd@seeder.com", password: "seedpassword")
+user.save!
 
-description = ["They have some signature dishes like pumpkin tortellini
-            and butternut squash ravioli that must be tried to be believed.",
-           "My husband had the butternut squash ravioli and it was seriously
-           the most amazing taste!",
-           "I recommend the Cacio e Pepe pizza and the baked ricotta appetizer,
-           hands down.",
-           "We had the Cacio e pepe Pizza and a Margherita Pizza, they were delicious!",
-           "The dining area was very clean and has a view into the open kitchen.",
-           "I went with here with my wife for our anniversary
-           and she loved it but i thought it was the worst place ever",
-           "The yuzu kosho aioli rocked, and the squid ink was a powerful flavor,
-           a great addition to the oyster.",
-           "Our server was absolutely incredible and even gave us powdered-foie
-           gras on the house.",
-           "(Lucky 8) Foie Gras, balsamic chocolate kabayaki, raisin cocoa pulp,
-           sip of aged sake.",
-           "My favorite dish is the lobster roll -while a bit pricy,
-           it is the best and I always get it",
-           "I bring my chihuahua here and she likes the steak tartar
-           the most, but personally i think its disgusting"]
+client = Yelp::Client.new({ consumer_key: ENV['CONSUMER_KEY'],
+                            consumer_secret: ENV['CONSUMER_SECRET'],
+                            token: ENV['TOKEN'],
+                            token_secret: ENV['TOKEN_SECRET']
+                          })
 
-ratings = [1, 2, 3, 4, 5]
 
-50.times do
+result = client.search('chinatown, ma' , { term: 'chinese' })
+result = JSON.parse(result.to_json)
+business_info = result['businesses']
+business_info.each do |business|
+  puts business['name']
   Restaurant.find_or_create_by!(
-    name: "#{name.sample}", street_address: "#{street_address.sample}",
-    city: "#{city.sample}", state: "#{state.sample}",
-    zip_code: "#{zip_code.sample}", description: "#{description.sample}",
-    phone: '012-345-6789', reservations: r_and_d.sample, delivery: r_and_d.sample
-    )
-end
-
-user = User.new(email: "seed@seeder.com", password: "seedpassword")
-user.save!
-
-Restaurant.all.each do |restaurant|
-  10.times do
-    restaurant.reviews.create(
-      body: "#{reviews.sample.capitalize}",
-      rating: ratings.sample, user_id: 1
-      )
-  end
+    name: "#{business['name']}", street_address: "#{business["location"]['address'][0]}",
+    city: "#{business["location"]['city']}", state: "Massachusetts",
+    zip_code: "#{business["location"]['postal_code']}", description: "#{business["snippet_text"]}",
+    phone: "#{business["display_phone"]}", reservations: r_and_d.sample, delivery: r_and_d.sample,
```

Comment:
Line is too long. [98/80]<br>Prefer single-quoted strings inside interpolations.